app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -112,10 +134,10 @@ function useUspsLocations() {
         address: bestMatchedAddress.address,
       });
     } else if (addressCandidates) {
-      fieldValidation?.current?.setCustomValidity(
+      validatedFieldRef?.current?.setCustomValidity(
         t('in_person_proofing.body.location.inline_error'),
       );
-      fieldValidation?.current?.reportValidity();
+      validatedFieldRef?.current?.reportValidity();
```

Comment:
No need for the `?` on `validatedFieldRef?` here - it will never be null/undefined. Only its `.current` value could be null/undefined.